File: yurt-jdbc/src/main/java/com/github/liulus/yurt/jdbc/annotation/If.java
package com.github.liulus.yurt.jdbc.annotation;

import java.lang.annotation.Documented;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * @author liulu
 * @version V1.0
 * @since 2020/11/10
 */
@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target({})
public @interface If {

    String value();

    String test() default "";

}

File: yurt-data/yurt-data-jdbc/src/main/kotlin/com/github/liulus/yurt/jdbc/annotation/If.kt
package com.github.liulus.yurt.jdbc.annotation

/**
 *
 * @author liulu
 * @version V1.0
 * @since 2020/11/3
 */
@Retention
@Target(AnnotationTarget.TYPE)
annotation class If(
        val value: String,
        val test: String = "",
)